CONTRIBUTING.md

```diff
@@ -1,11 +1,39 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated!
+Every little bit helps, and credit will always be given.
+
 ## Contributing Guidelines
 
-- If you'd like to make a pull request to contribute, please keep it simple. If you use some higher level techniques I don't understand, I'm not going to approve it because I won't be able to maintain it.
+- If you'd like to make a pull request to contribute, please ***keep it simple***. If you use some higher level techniques I don't understand, I'm not going to approve it because I won't be able to maintain it.
 
-- Please don't make a pull request with a bunch of changes in syntax just for the sake of 'best practices' (ex: changing "if blah == False" to "if not blah"). Unless something makes a difference to performance frankly I don't care.
+- *Please* ensure all code has been formatted by [Black](https://pypi.org/project/black/) ([Usage Guide](https://black.readthedocs.io/en/stable/getting_started.html#basic-usage)). This helps to maintain code readability and keep a consistent style across all files. This is optional for you to do.
```

Comment:
```suggestion
- *Please* ensure all code has been formatted by [Black](https://pypi.org/project/black) ([Usage Guide](https://black.readthedocs.io/en/stable/getting_started.html#basic-usage)). This helps to maintain code readability and keep a consistent style across all files. This is optional for you to do.
```

remove extra slash

Reply:
That **is** the url though, without the slash autocorrects when entered.

Reply:
> That **is** the URL though, without the slash it autocorrects when entered.

wdym?

Reply:
When you enter ` https://pypi.org/project/black` into the URL bar it is forwarded to `https://pypi.org/project/black/`

![image](https://user-images.githubusercontent.com/34950371/177856397-bcf847e4-1848-4351-9188-711688a00ea4.png)

Therefore I'd suggest that the URL with a `/` is the correct URL as the developers of the site would like.